operator/api/v1alpha1/central_types.go

```diff
@@ -611,6 +615,10 @@ const (
 	ScannerV4ComponentDisabled ScannerV4ComponentPolicy = "Disabled"
 )
 
+type ConfigAsCodeSpec struct {
+	Enabled *bool `json:"enabled,omitempty"`
```

Comment:
General remark: I think we should try to aim for consistency -- at least for new features. In the recent past I think there was a tendency to use the reversed logic and add `disabled` switches (e.g. for Scanner V4) instead of `enabled` switches. WDYT?

Reply:
Ha, I did consider whether it should be `enabled` or `disabled`.  I went with `enabled` because it's my preference and also because it wasn't clear to me looking at the values template that there was a consistent pattern.

Taking a look again, I see these uses:

`disable`: scanner, scannerv4, autoscaling
`enabled`: monitoring, telemetry, exposure

That said, I'm okay to change it if you want to make `disabled` the established pattern.

Reply:
Just to follow up here:  My preference would be to keep `enabled`, but if you would still prefer me to switch to `disable`, I am happy to do so.

Reply:
Personally, I am not leaning to any side here. It's just that I believe we should (1) accept that we haven't been consistent in the past and (2) nevertheless strive for more consistency on our path forward. And the most recent example of an addition that I had in mind was the `scannerV4.disabled` toggle (there might be others). So that would be an indicator for switching to `disabled`-style toggles.

I would also be fine with bringing this up in some eng Slack channel in order to increase visibility on this matter. WDYT?

Reply:
Alright, don't treat this as a blocker from my side. Personally I would also be in favor of the positive form. Thank you for bringing it up on Slack -- looks like at least among those who participated in the thread, there is some tendency for the positive form as well.